pipelines/services/utils.go

```diff
@@ -11,7 +10,6 @@ import (
 func constructPipelinesURL(qParams map[string]string, apiURL, apiPath string) (string, error) {
 	uri, err := url.Parse(apiURL + apiPath)
 	if err != nil {
-		log.Error("Failed to parse pipelines fetch run status url")
```

Comment:
See my other comment in this CR cycle. It may apply here as well.

Reply:
This should be probably deleted already in my previous commit